hikari/applications.py

```diff
@@ -272,7 +270,7 @@ class OwnConnection:
     """The visibility of the connection."""
 
 
-@attr.define(hash=True, kw_only=True, weakref_slot=False)
+@attr.define(frozen=True, hash=True, kw_only=True, weakref_slot=False)
```

Comment:
Instead of this, could we use `attr.frozen`? I was thinking about this when we changed to the next gen API

Reply:
sure